python/test/unit/runtime/test_subproc.py

```diff
@@ -70,14 +81,23 @@ def kernel_dot(Z):
 
 def test_compile_in_forked_subproc() -> None:
     reset_tmp_dir()
-    major, minor = torch.cuda.get_device_capability(0)
-    cc = major * 10 + minor
+    try:
+        import torch
+    except ImportError:
+        raise ImportError("Triton requires PyTorch to be installed")
+    if torch.version.hip is None:
+        major, minor = torch.cuda.get_device_capability(0)
+        cc = major * 10 + minor
+        device_type="cuda"
+    else:
+        cc = None
+        device_type="hip"
```

Comment:
LGTM. 
Minor comment: This code seems to be identical to the one above (line 41). Maybe we can abstract it into a function called set_device_type() to reduce code duplication.